libbeat/otelbeat/oteltranslate/outputs/elasticsearch/config_otel.go

```diff
@@ -124,20 +125,32 @@ func ToOTelConfig(output *config.C, logger *logp.Logger) (map[string]any, error)
 		"timeout":           escfg.Transport.Timeout,         // timeout
 		"idle_conn_timeout": escfg.Transport.IdleConnTimeout, // idle_connection_timeout
 
+		// For libbeat ES output, the "workers" setting controls the number of concurrent connections per ES host.
+		// For elasticsearchexporter, we can achieve the same concurrency by specifying "max_conns_per_host" setting
+		// as our otelconsumer sends data parallelly to the consumer.
+		// Also, we use http/1 in libbeat. To achieve parity, disable force_attempt_http2
+		"max_conns_per_host":  escfg.NumWorkers(),
+		"force_attempt_http2": false,
+
 		// Retry
 		"retry": map[string]any{
 			"enabled":          true,
 			"initial_interval": escfg.Backoff.Init, // backoff.init
 			"max_interval":     escfg.Backoff.Max,  // backoff.max
 			"max_retries":      escfg.MaxRetries,   // max_retries
-
 		},
 
-		// Batcher is experimental
-		"batcher": map[string]any{
-			"enabled":  true,
-			"max_size": escfg.BulkMaxSize, // bulk_max_size
-			"min_size": 0,                 // 0 means immediately trigger a flush
+		"sending_queue": map[string]any{
+			"batch": map[string]any{
+				"max_size": escfg.BulkMaxSize, // bulk_max_size
+				"min_size": 0,                 // 0 means immediately trigger a flush
+				"sizer":    "items",
+			},
+			"enabled":           true,
+			"queue_size":        getQueueSize(logger, output),
+			"block_on_overflow": true,
+			"wait_for_result":   true,
+			"num_consumers":     escfg.NumWorkers(),
```

Comment:
What is the reason that upstream allows this to be decoupled from the number of connections? In Beats it isn't, a worker is a consumer is a connection, so this matches, but I am wondering if there is a performance implication here where we might not always want them to match in the collector.

Reply:
Thanks, can you add this context as a comment for why we did this?